face/src/plugins/timeline/TimelineView.js

```diff
@@ -120,13 +172,50 @@ class TimelineView extends BaseChartView {
         }
         bars.attrs({
             x: d => this.x(d.start),
-            width: d => this.x(Math.max(d.stop - d.start))
+            width: d => this.x(d.stop) - this.x(d.start)
         });
-        this.makeLeftAxis(group.append('g').classed('chart__axis', true), {
-            tickSize: 0,
-            tickFormat: '',
-            scale: y
+        return height;
+    }
+
+    drawMiniHost(host, offset) {
```

Comment:
Care to avoid duplicated code and create some helper functions to draw both main and mini chart?